internal/pod_client.go

```diff
@@ -246,6 +246,9 @@ func (client *realFdbPodClient) GetVariableSubstitutions() (map[string]string, e
 	}
 	substitutions := map[string]string{}
 	err = json.Unmarshal([]byte(contents), &substitutions)
+	if err != nil {
+		log.Error(err, "Error deserializing pod substitutions", "responseBody", contents)
```

Comment:
Shouldn't the caller method actually log the error? Otherwise we log the same error multiple times.

Reply:
It logs the error, but JSON parsing errors don't provide much context, so I think logging it twice will be helpful.